plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoCreate.ts

```diff
@@ -122,6 +126,8 @@ export function createGithubRepoCreateAction(options: {
         deleteBranchOnMerge = false,
         allowMergeCommit = true,
         allowSquashMerge = true,
+        squashMergeCommitTitle = 'PR_TITLE',
+        squashMergeCommitMessage = 'COMMIT_MESSAGES',
```

Comment:
Is this a new default behavior, or is this how it was before? If it's new I think we can mention it in the changelog.

Reply:
Good eye! I thought I'd matched the default, but a careful read of materials ([blog post](https://github.blog/changelog/2022-08-23-new-options-for-controlling-the-default-commit-message-when-merging-a-pull-request/) tells me the default is "default message," described as:

> Default message: commit title and commit message (if the pull request contains a single commit), or the pull request title and number and list of commits (if the pull request contains multiple commits)

This tells me the default commit title is COMMIT_OR_PR_TITLE and the message is COMMIT_MESSAGES.

I'll change things. Alternatively, being that this field is optional, I could leave the default undefined and rely on the APIs defaults. I didn't see prior art for this in other parameters though. Let me know what backstage prefers in these circumstances.

Reply:
Updated the default to COMMIT_OR_PR_TITLE.

Again, let me know if I should leave the default as undefined and rely on the underlying APIs defaults instead.